src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -158,6 +159,11 @@ def __init__(
 
         self._siblings = []
 
+        if not ast_stmt:
+            ast_stmt = pd.session._ast_batch.assign()
```

Comment:
I don't really understand the point of creating an `assign` node like this. If a statement isn't actually supplied, how a "default" one won't really work. Is this because of the way that Pandas interacts with this code?